lessons/beginners/print/index.md

```diff
@@ -146,7 +146,7 @@ na výrazy a jejich hodnoty podíváme podrobněji.
 > print(1, 2, 3)
 > ```
 >
-> Kolem operátorů jako `+` a `*` se obyčejně píše jedna mezera zleva a
+> Kolem operátorů jako `+` a `/` se obyčejně píše jedna mezera zleva a
```

Comment:
Just curious, what is the reason for this particular change?

Reply:
To match the example just below

Reply:
... and as to why use that example, ½ looks tighter than 2·2 or other expressions I could find, so it looks best without spaces.